### ISSUE TITLE
"rails generate friendly_id" does not produce correct migration syntax for Rails 5.1

### ISSUE THREAD

**Issue by chrickso** [norman/friendly_id#817]
Hi. I'm new to Rails so apologies if this is something on my end done wrong but I am following a tutorial that has me installing friendly_id and after following the friendly_id documentation instructions of running "rails generate friendly_id", the db:migrate is failing with the following message:

```
Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class CreateFriendlyIdSlugs < ActiveRecord::Migration[4.2]
```

i got it working by adding "[5.1]" after the ActiveRecord::Migration in the migration file and rerunning db:migrate. I assume this is due to something new in Rails 5.1 because my instructor was running 5.0 and didn't have any issues.

```
class CreateFriendlyIdSlugs < ActiveRecord::Migration[5.1]
  def change
    create_table :friendly_id_slugs do |t|
      t.string   :slug,           :null => false
      t.integer  :sluggable_id,   :null => false
      t.string   :sluggable_type, :limit => 50
      t.string   :scope
      t.datetime :created_at
    end
    add_index :friendly_id_slugs, :sluggable_id
    add_index :friendly_id_slugs, [:slug, :sluggable_type]
    add_index :friendly_id_slugs, [:slug, :sluggable_type, :scope], :unique => true
    add_index :friendly_id_slugs, :sluggable_type
  end
end
```

hope this helps. thanks!

**Comment by lazyatom** [comment]
This is addressed in #816, I hope

**Comment by superkunn** [comment]
Than Q very much

**Comment by olivierlacan** [comment]
@lazyatom @chrickso Can this issue be closed?

**Comment by lazyatom** [comment]
Yes, it was fixed in #858. Thanks!
